### ISSUE TITLE
Collaboration with CNCF The End User Technical Advisory Board

### ISSUE THREAD

**Issue by svrnm** [open-telemetry/sig-end-user#50]
At KCD Munich I attended a presentation of the newly formed CNCF The End User Technical Advisory Board:

https://www.cncf.io/people/end-user-technical-advisory-board/

From my understanding they share a lot of goals and responsibilities with our End User SIG. I think both groups should see how they can work together best. @alolita: as the chair of the TAB can you share with the End User SIG if and how such a collaboration could look like, or if it is too early for that with the TAB still being in it's early phase?

**Comment by danielgblanco** [comment]
I agree. We will all benefit from joining forces and collaborate with this TAB to shape the scope of this SIG. I think it may also be useful for this SIG to provide info to the TAB on the initiatives we run and how they perform.

**Comment by alolita** [comment]
@svrnm the CNCF End User TAB interacts with the larger End User community. There are multiple workgroups within the TAB as well as the end user SIG. Let's sync to figure out a list of areas we (OTel) are looking at as actionable areas of collaboration with the end user community.

**Comment by svrnm** [comment]
> Let's sync to figure out a list of areas we (OTel) are looking at as actionable areas of collaboration with the end user community.

thanks, this is what I was thinking about.
